components/avatar/index.css

```diff
@@ -10,107 +10,126 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+  --spectrum-avatar-inline-size: var(--spectrum-avatar-size-200);
+  --spectrum-avatar-block-size: var(--spectrum-avatar-size-200)
 }
 
 .spectrum-Avatar--size300 {
-  @remapvars {
-    find: /--spectrum-avatar-size-300(.*)/;
-    filter: color;
-    replace: --spectrum-avatar$1;
-  }
+  --spectrum-avatar-inline-size: var(--spectrum-avatar-size-300);
+  --spectrum-avatar-block-size: var(--spectrum-avatar-size-300)
 }
 
 .spectrum-Avatar--size400 {
-  @remapvars {
-    find: /--spectrum-avatar-size-400(.*)/;
-    filter: color;
-    replace: --spectrum-avatar$1;
-  }
+  --spectrum-avatar-inline-size: var(--spectrum-avatar-size-400);
+  --spectrum-avatar-block-size: var(--spectrum-avatar-size-400)
 }
 
 .spectrum-Avatar--size500 {
-  @remapvars {
-    find: /--spectrum-avatar-size-500(.*)/;
-    filter: color;
-    replace: --spectrum-avatar$1;
-  }
+  --spectrum-avatar-inline-size: var(--spectrum-avatar-size-500);
+  --spectrum-avatar-block-size: var(--spectrum-avatar-size-500)
 }
 
 .spectrum-Avatar--size600 {
-  @remapvars {
-    find: /--spectrum-avatar-size-600(.*)/;
-    filter: color;
-    replace: --spectrum-avatar$1;
-  }
+  --spectrum-avatar-inline-size: var(--spectrum-avatar-size-600);
+  --spectrum-avatar-block-size: var(--spectrum-avatar-size-600)
 }
 
 .spectrum-Avatar--size700 {
-  @remapvars {
-    find: /--spectrum-avatar-size-700(.*)/;
-    filter: color;
-    replace: --spectrum-avatar$1;
+  --spectrum-avatar-inline-size: var(--spectrum-avatar-size-700);
+  --spectrum-avatar-block-size: var(--spectrum-avatar-size-700)
+}
+
+@media (forced-colors: active) {
+  .spectrum-Avatar {
+    --highcontrast-avatar-color-opacity-disabled: 1;
+    --highcontrast-avatar-focus-indicator-color: Highlight;
   }
 }
 
 .spectrum-Avatar {
   display: inline-block;
   position: relative;
-  inline-size: var(--spectrum-avatar-width);
-  block-size: var(--spectrum-avatar-height);
+  inline-size: var(--mod-avatar-inline-size, var(--spectrum-avatar-inline-size));
+  block-size: var(--mod-avatar-block-size, var(--spectrum-avatar-block-size));
 
-  border-radius: var(--spectrum-avatar-border-radius);
+  border-radius: var(--mod-avatar-border-radius, var(--spectrum-avatar-border-radius));
   border-width: 0;
 
+  outline: none;
+
   -webkit-user-drag: none;
   user-select: none;
-  overflow: hidden;
 
-  &:after {
-    content: '';
+  overflow: visible;
+
+  opacity: var(--mod-avatar-color-opacity, var(--spectrum-avatar-color-opacity));
+
+  &.is-disabled {
+    opacity: var(--highcontrast-avatar-color-opacity-disabled, var(--mod-avatar-color-opacity-disabled, var(--spectrum-avatar-color-opacity-disabled)));
+  }
+
+  &:focus-within:not(.is-disabled) .spectrum-Avatar-link::after {
+    pointer-events: none;
+    content: "";
     position: absolute;
-    inset-block-start: 0;
-    inset-block-end: 0;
-    inset-inline-start: 0;
-    inset-inline-end: 0;
-    box-sizing: border-box;
-    border-width: var(--spectrum-avatar-border-size);
+    top: calc(
```

Comment:
This should be

```
    top: calc((var(--mod-avatar-focus-indicator-gap, var(--spectrum-avatar-focus-indicator-gap)) 
              + var(--mod-avatar-focus-indicator-thickness, var(--spectrum-avatar-focus-indicator-thickness))) * -1);

```

So that if either the border width or the focus indicator gap ever change, the indicator will still center with the component.

Also, I think it _might_ be better to use margin-block-start instead of "top".

Reply:
> Also, I think it might be better to use margin-block-start instead of "top".

Good call-out! I looked into this a little, and I think it's `inset-block-start`: https://developer.mozilla.org/en-US/docs/Web/CSS/inset-block-start